miscelanea/003-scc-and-caps/poc-003.yaml

```diff
@@ -0,0 +1,563 @@
+            - "SETFCAP"
+            # - "FSETID"
+            # - "NET_RAW"
+            # - "SYS_CHROOT"
+
+            # No default capabilities
+            - "SYS_ADMIN"
+            - "SYS_RESOURCE"
+            # - "MKNOD"
+
+      command: ["/usr/sbin/init"]
+      args:
+        - exit-on-finished
+        # - no-exit
+        - ipa-server-install
+        - -U
+        # - --hostname
+        # - poc-003.apps.permanent.idmocp.lab.eng.rdu2.redhat.com
+        - --realm
+        - apps.permanent.idmocp.lab.eng.rdu2.redhat.com
+        - --no-ntp
+        - --no-sshd
+        - --no-ssh
+        - --verbose
+      env:
+        # - name: SYSTEMD_OPTS
+        #   value: "--show-status=true --unit=ipa-server-configure-first.service"
+        - name: KRB5_TRACE
+          value: /dev/console
+        - name: SYSTEMD_LOG_LEVEL
+          value: "debug"
+        - name: SYSTEMD_LOG_TARGET
+          value: "console"
+        - name: SYSTEMD_LOG_COLOR
+          value: "no"
+        - name: INIT_WRAPPER
+          value: "1"
+        - name: DEBUG_TRACE
+          valueFrom:
+            configMapKeyRef:
+              name: poc-003
+              key: DEBUG_TRACE
+        - name: PASSWORD
+          valueFrom:
+            configMapKeyRef:
+              name: poc-003
+              key: PASSWORD
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: UID_BASE
+          valueFrom:
+            configMapKeyRef:
+              name: poc-003
+              key: UID_BASE
+        - name: GID_BASE
+          valueFrom:
+            configMapKeyRef:
+              name: poc-003
+              key: GID_BASE
+        - name: IPA_SERVER_HOSTNAME
+          valueFrom:
+            configMapKeyRef:
+              name: poc-003
+              key: IPA_SERVER_HOSTNAME
+        - name: container_uuid
+          valueFrom:
+            configMapKeyRef:
+              name: poc-003
+              key: container_uuid
+        # - name: IPA_SERVER_IP
+        #   valueFrom:
+        #     fieldRef:
+        #       fieldPath: status.podIP
+
+        - name: container
+          valueFrom:
+            configMapKeyRef:
+              name: poc-003
+              key: container
+        - name: SYSTEMD_OFFLINE
+          value: "1"
+        - name: SYSTEMD_NSPAWN_API_VFS_WRITABLE
+          value: "network"
+
+      volumeMounts:
+        - name: data
+          mountPath: /data
+        # - name: systemd-run-rpcbind
+        #   mountPath: /run/rpcbind
+        - name: systemd-tmp
+          mountPath: /tmp
+        - name: systemd-sys
+          mountPath: /sys
+          readOnly: true
+        - name: systemd-sys-fs-selinux
+          mountPath: /sys/fs/selinux
+          readOnly: true
+        - name: systemd-sys-firmware
+          mountPath: /sys/firmware
+          readOnly: true
+        - name: systemd-sys-kernel
+          mountPath: /sys/kernel
+          readOnly: true
```

Comment:
This is one block to investigate for removing `allowHostDirVolumePlugin: true` from the SCC object.